### DISCUSSION TITLE
Guess the artist 🖼️🎨🖌️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1249] [Galaxy Trivia] [answered]
![image](https://github.com/ParanoidUser/yolo/assets/5120290/d4f7bd74-fe29-471f-bb58-3da99d47d605)

**Comment by peterbaccaro** [comment]
The artist **Eric Deper**.

**Comment by danielcshn** [comment, marked as answer]
The artist **Piet Mondrian**.

**Reply by ParanoidUser** [reply]
Good eye, Daniel! 👏

**Comment by KR-Ravindra** [comment]
Salvador Dalí, the master of surrealism
